src/test/javascript/socketio.js

```diff
@@ -88,6 +88,7 @@ modes.forEach(function (mode) {
       });
       socket.on("m", function(arg, ack) {
         ack("This is an acked ack");
+        done();
```

Comment:
We getting errors with the following message: "Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves."

I'm not familiar with how socket.io test library works, so I'm just guessing this here. Let's how it goes.

Reply:
Hmm, maybe calling `done` is not required when using `ack` and only when running non-test actions. Like disconnecting (see bellow).